open-metadata-implementation/governance-servers/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerVirtualizationService.java

```diff
@@ -56,14 +66,76 @@ public VoidResponse setVirtualizerConfig(String userId, String serverName, Virtu
                     )
             );
 
+        } catch (Exception e){
+            log.error("Invalid configuration! ", e);
+        }
+
+        /**
+         * Set up connection to the virtualization service
+         */
+
+        Map<String, Object> additionalProperties = new HashMap<>();
+        Endpoint endpoint = new Endpoint();
+
+        final String endpointGUID             = UUID.randomUUID().toString();
+        final String connectionGUID           = UUID.randomUUID().toString();
+        final String endpointDescription      = "Virualization solution endpoint.";
+        final String connectionDescription    = "Virualization solution connection.";
+
+        String endpointName    = "Virtualizer.Endpoint." + serverName;
+
+        if ("org.odpi.openmetadata.openconnectors.governancedaemonconnectors.viewgenerator.virtulaizationsolutionconnector.derby".equals(virtualizationConfig.getVirtualizationProvider())){
```

Comment:
Creation of connection object during configuration should be done in the configuration factory.